### ISSUE TITLE
Newcomer Install

### ISSUE THREAD

**Issue by 3kynox** [Automattic/wp-calypso#7398]
Hello,

Newcomer first (surely) dumb question : 

![](https://puu.sh/qwUU4/88713dc0a2.png)

What's missing please ?

Debian Jessie Box here.

**Comment by aduth** [comment]
Hi @3kynox 

Could you try the following troubleshooting steps:

- Switch to the desired Node version using `n 6.1.0` (noting that you've installed `n` already)
- Ensure you have the latest master by running `git pull`

Afterward, run `make run` again and let me know if you're still having issues.

**Comment by 3kynox** [comment]
Thanks for fast answer, unfortunately, no changes : 

![](https://puu.sh/qwW2I/1a88bcd392.png)

**Comment by aduth** [comment]
Could you try `make distclean && make run` to reinstall all dependencies?

cc also @giuseppeg for guidance on usage stats script (on which it's bailing)

**Comment by giuseppeg** [comment]
Oh hai. Not quite sure what's happening. 
From the screenshot it seems that it is failing at requiring `async`. 
Did something change in the way how you bundle code or the webpack conf in general?
That's the first place where I would look for issues if `make distclean` doesn't work.

If not maybe you can temporarily replace and "mock" async to see if it fails at requiring other deps too.
```javascript
var async = {
  each: function (a,b,c) {
     c()
  }
}
```

Btw I'd remove that script completely if you decide not to merge #5481

**Comment by 3kynox** [comment]
Hello again,

sorry guys, just slept in front of keyboard, it was late in France :)

make distclean did the job, currently node package install is done.

**Comment by 3kynox** [comment]
hmm.. configured host file and local browser to connect to my VPS as a proxy.

http://calypso.localhost:3000 works great with chrome but not with FF.
FF looks like to rename all the time http://calypso.localhost:3000 to http://www.calypso.localhost:3000

Also any directive for calypso to work with apache vhost configuration file ?

Thanks !

**Comment by 3kynox** [comment]
I tried put in my config a little modification of what I've seen here : https://github.com/Automattic/wp-calypso/issues/666

<VirtualHost *:3000>
ServerName noxen.net
ProxyPreserveHost On
ProxyPass / http://calypso.localhost:3000/
ProxyPassReverse / http://calypso.localhost:3000/
</VirtualHost>

But don't looks like to work, just display WP logo in the middle of page.

**Comment by aduth** [comment]
I'm not terribly familiar with Apache configurations for Calypso.

Could you check to see if there are any errors in your browser developer tools console, or failed requests on the developer tools Network tab?

Also ensure that you're logged in to WordPress.com before visiting Calypso locally (though I believe it should be prompting you to log in if you're not already)

**Comment by 3kynox** [comment]
No errors on console, it looks like wrong adress is entered. it's explained on wiki adress used must be http://calypso.localhost:3000 6 if just localhost is used, we only have wp logo in the middle of page. Behavior is the same here.

Little update, I've added a vhost adress : http://calypso.noxen.net:3000 and now configuration is : 

```
<VirtualHost *:3000>
ServerName calypso.noxen.net
ProxyPreserveHost On
ProxyPass / http://127.0.0.2:3000/
ProxyPassReverse / http://127.0.0.2:3000/
</VirtualHost>
```

I also added 127.0.0.2 calypso.noxen.net to host file (previous entry 127.0.0.2 calypso.localhost still here too).

And launched make run, no changes currently.
